### DISCUSSION TITLE
AG 1.2.0 beta5

### DISCUSSION THREAD

**Discussion by fojerhar** [apohl79/audiogridder#720] [General]
Hi

I have installed excitedly the new beta5 version, and I had the message from pro tools, that this is not a valid 64bit aax plugin. Tried both x86_64 and the universal installer too.
System: AMD hack with catalina, PT 2021.7
<img width="653" alt="Screenshot 2021-09-21 at 12 06 49" src="https://user-images.githubusercontent.com/72139943/134158078-f3dbe73a-e6f2-47ea-9f81-1daace7b8516.png">

thank you

**Comment by apohl79** [comment]
Ok, something must have gone wrong with the AAX signing step. I will check the build.

**Reply by apohl79** [reply]
confirmed, I'm sorry. will provide new builds shortly.

**Reply by fojerhar** [reply]
You should not sorry, you are making an amazing thing, big thank you

**Reply by apohl79** [reply]
Ok I need to take that back. Only my archive did fail, which is not important at all...

I tested the macOS installers - both x86 and universal (which I assumed pro tools does not support yet), all working fine on Pro Tools 2020.12.0.

Try removing all AudioGridder plugins from "/Library/Application Support/Avid/Audio/Plug-Ins/" and install again.

**Reply by fojerhar** [reply]
HI
Uninstalled, and only ag instr and midi shows up, and they are seems working.
<img width="647" alt="Screenshot 2021-09-23 at 12 03 37" src="https://user-images.githubusercontent.com/72139943/134489341-48d4c401-0c5c-440b-bd76-fbac5bb64649.png">

**Reply by Wujian-sinemedia** [reply]
I have same error in my own built. please give more detail for "not a valid 64bit aax plugin."

**Reply by apohl79** [reply]
> I have same error in my own built. please give more detail for "not a valid 64bit aax plugin."

You have to sign AAX plugins to make pro tools load them.
